[BE][profiler] add test for EventList

davidberard98 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #113764

EventList isn't really tested in CI because it seems to only really be used when kineto is not available.

Add a basic sanity test that would have caught #113756

davidberard98 committed:
[profiler] add test for EventList

EventList isn't really tested in CI because it seems to only really be used when kineto is not available.

Add a basic sanity test.

[ghstack-poisoned]

malfet reviewed (approved):
Yay, it fails, let me `merge -f` my change and rebase

malfet commented:
@pytorchbot rebase -b main

davidberard98 committed:
Update on "[BE][profiler] add test for EventList"


EventList isn't really tested in CI because it seems to only really be used when kineto is not available.

Add a basic sanity test that would have caught #113756

[ghstack-poisoned]

davidberard98 commented:
@pytorchbot merge

davidberard98 commented:
@pytorchbot merge
